src/main/java/com/faforever/client/patch/GameUpdaterImpl.java

```diff
@@ -99,6 +95,19 @@ public GameUpdater addFeaturedModUpdater(FeaturedModUpdater featuredModUpdater)
             createFaPathLuaFile(initFile.getParent().getParent());
             copyLegacyInitFile(initFile);
           }
+        })
+        .exceptionally(throwable -> {
+          boolean allowReplaysWhileInGame = preferencesService.getPreferences().getForgedAlliance().isAllowReplaysWhileInGame();
+          if (throwable.getCause().getCause() instanceof AccessDeniedException && allowReplaysWhileInGame) {
+            log.info("Unable to update file cause they are not writeable. " +
+                "Experimental feature is turned on " +
+                "that allows multiple game instances to run in parallel this is most likely the cause.");
+            throw new UnsupportedOperationException("Unable to patch Forged Alliance to the required version due to conflicting version running");
```

Comment:
please always pass on previous Exceptions
```suggestion
            throw new UnsupportedOperationException("Unable to patch Forged Alliance to the required version due to conflicting version running", throwable);
```

Reply:
also don't we want to display a meaningful Exception

Reply:
what do you mean? To the user? That is done in an exceptionally

Reply:
Have you ever seen 
```
RuntimeException at X 
   at ...
   at ....
Caused by RealException: the message you are really looking for
   at ....
```
That happens if you pass on the causing Exception to a new on e thrown like here, what I suggested....

Reply:
https://www.baeldung.com/java-wrapping-vs-rethrowing-exceptions

Reply:
Also pass Exception to info log statement I guess

Reply:
What if there is an unexpected Exception it is never logged we will never know what throwable really contained

Reply:
ah yeah got it just misunderstood